providers/implementations/rands/seeding/rand_cpu_x86.c

```diff
@@ -35,9 +40,36 @@ size_t prov_acquire_entropy_from_cpu(RAND_POOL *pool)
 
     bytes_needed = rand_pool_bytes_needed(pool, 1 /*entropy_factor*/);
     if (bytes_needed > 0) {
+
         buffer = rand_pool_add_begin(pool, bytes_needed);
 
         if (buffer != NULL) {
+#if defined(OPENSSL_SYSNAME_TANDEM) && defined(_TNS_X_TARGET)
```

Comment:
Wouldn't it make sense to have this as a separate function?

Reply:
> 
> 
> Wouldn't it make sense to have this as a separate function?

Probably. I can reissue the pull request with this change. I was also concerned that the location of the change may have been inappropriate. If it is proper here, I can make this happen tomorrow.

Reply:
@paulidale or @mspncp ?

Reply:
Refactored randomizer fetch code into separate method and pushed.